packages/components/src/components/domainproperties/ConditionalFormattingAndValidation.tsx

```diff
@@ -184,7 +184,7 @@ export class ConditionalFormattingAndValidation extends React.PureComponent<Cond
                 <Row className='domain-row-expanded'>
                     <Col xs={12}>
                         {!hideConditionalFormatting && this.renderConditionalFormats()}
-                        {this.renderValidator(false)}
+                        {field.dataType.isString() && this.renderValidator(false)}
```

Comment:
Minor but can we add in a `DomainField.hasRegExValidation(field)` helper for this to match the next line for the range validation? Then in the DomainField that hasRegExValidation can `return field.dataType.isString()`.

Reply:
Good call! Fixed